packages/python/test/integration-setup.js

```diff
@@ -28,19 +28,12 @@ module.exports = function setupTests(groupIndex) {
 
   // eslint-disable-next-line no-restricted-syntax
   for (const fixture of chunkedFixtures) {
-    // Python endpoints currently require the AL2 build image
-    const projectSettings = {
-      nodeVersion: '18.x',
-    };
-
     const errMsg = testsThatFailToBuild.get(fixture);
     if (errMsg) {
       // eslint-disable-next-line no-loop-func
       it(`should fail to build ${fixture}`, async () => {
         try {
-          await testDeployment(path.join(fixturesPath, fixture), {
-            projectSettings,
```

Comment:
What does removing `projectSettings` do?

Reply:
Or rather, why are we removing it?

Reply:
It was here before to force all projects to be at node version 18. In this PR, Nate changed the default to be whatever the platform default is, which means we don't need to specify the project setting here.

The platform default is node Node 20.

Reply:
We were forcing the AL2 build image previously, since Python was not yet supported. Removing project settings here will use the default build image, which is now the AL2023 image.